templates/common/gcp/files/usr-local-bin-update-dns-server.yaml

```diff
@@ -0,0 +1,22 @@
+mode: 0755
+path: "/usr/local/bin/update-dns-server"
+contents:
+  inline: |
+    #!/bin/bash
+    # This is test code
+    
+    echo "Executing update-dns-server"
+
+    if ![-f /etc/NetworkManager/conf.d]; then
+      mkdir /etc/NetworkManager/conf.d
+      touch /etc/NetworkManager/conf.d/dns-servers.conf
+    fi
+    if ![-f /etc/NetworkManager/conf.d/dns-servers.conf]; then
+        touch /etc/NetworkManager/conf.d/dns-servers.conf
+    fi
+
+    echo "Added by OpenShift" >> /etc/NetworkManager/conf.d/dns-servers.conf
```

Comment:
I assume this is intended to be a comment.

Also, I think this one should be > not >>. Otherwise this is all going to append to the file every time the node reboots since this is a persistent file.

Reply:
Correct on both. Updated.